low-level/files/opt/seagate/sspl/setup/resource_map/lr2/error.py

```diff
@@ -25,3 +25,11 @@ def __init__(self, rc=0, desc=None, message_id=None, message_args=None):
         """Initialize ResourceMapError."""
         super(ResourceMapError, self).__init__(
               rc, 'SSPL: %s' % desc, message_id, message_args)
+
+class ManifestError(BaseError):
```

Comment:
Why resource map Error wont serve Manifest too, like its been done for health. What special been done here anyways ?

Reply:
Corrected.